openedx/adg/lms/applications/admin.py

```diff
@@ -408,6 +409,7 @@ def changeform_view(self, request, object_id=None, form_url='', extra_context=No
                     self._save_admin_notes(application, user, note)
 
                 send_application_status_update_email(application, message_for_applicant)
+                emit_application_progress(application.user)
```

Comment:
We should emit this signal in the applications dashboard code as well, that we have written for Phase-II. We may not be able to do the same for webinars atm, since the webinars implementation for the phase-II has not been done yet, but the application review page integration is completed. When an admin makes a decision, an emit signal should be fired.

Reply:
For webinar related, we can keep a note and later do it once its implementation is being done (if required).

Reply:
Got it, incooperating these changes too!